nix/modules/landing-page/static/custom.js

```diff
@@ -0,0 +1,27 @@
+document.addEventListener('DOMContentLoaded', (event) => {
+    openDetailsIfHashMatches();
+});
+
+window.addEventListener('hashchange', (event) => {
+    openDetailsIfHashMatches();
+});
+
+function openDetailsIfHashMatches() {
+    const hash = window.location.hash;
```

Comment:
```js
if (!hash) return;
const targetElement = document.querySelector(hash);
if (!targetElement) return;
let parent = targetElement.parentElement;
while (parent) {
  if (parent.tagName.toLowerCase() === 'details' && !parent.open) {
      parent.open = true;
      break;
  }
  parent = parent.parentElement;
}
requestAnimationFrame(() => {
    targetElement.scrollIntoView({ behavior: "smooth", block: "start" });
});
```

Reply:
That's definitely an improvement, thank you! (Applied your suggestion)